tools/scripts/generate_binary_build_matrix.py

```diff
@@ -117,6 +119,35 @@ def translate_desired_cuda(gpu_arch_type: str, gpu_arch_version: str) -> str:
 def list_without(in_list: List[str], without: List[str]) -> List[str]:
     return [item for item in in_list if item not in without]
 
+def get_conda_install_command(channel: str, gpu_arch_type: str, arch_version: str) -> str:
+    conda_channels = "-c pytorch" if channel == "release" else f"-c pytorch-{channel}"
+
+    if gpu_arch_type == "cuda":
+        if arch_version == "10.2" or arch_version == "11.3":
```

Comment:
Should we just do a numerical comparison here?

```suggestion
        if float(arch_version) <= 11.3:
```